pkg/notify/notifier/dingtalk/dingtalk.go

```diff
@@ -119,6 +139,11 @@ func NewDingTalkNotifier(logger log.Logger, receivers []config.Receiver, notifie
 			n.templateName = opts.Global.Template
 		}
 
+		if len(d.TmplType) > 0 && strings.Compare("markdown", d.TmplType) == 0 {
```

Comment:
You can refer to [this](https://github.com/kubesphere/notification-manager/blob/master/pkg/notify/notifier/dingtalk/dingtalk.go#L116) to implement the following logic.

- If the global DingTalk TmplType is not set and the local DingTalk TmplType is set, the local DingTalk TmplType will be used.
- If the global DingTalk TmplType is set and the local DingTalk TmplType is set too, the local DingTalk TmplType will be used.
- If the global DingTalk TmplType is set and the local DingTalk TmplType is not set, the global DingTalk TmplType will be used.

Reply:
this is logic to set global DingTalk TmplType  from config, default is text.